frameworks/Java/hserver/src/main/java/com/test/hserver/controller/TestController.java

```diff
@@ -0,0 +1,24 @@
+package com.test.hserver.controller;
+
+import com.test.hserver.bean.Message;
+import top.hserver.core.ioc.annotation.Controller;
+import top.hserver.core.ioc.annotation.GET;
+
+/**
+ * @author hxm
+ */
+@Controller
+public class TestController {
+
+    private static Message message = new Message();
+
+    @GET("/json")
+    public Message json() {
+        return message;
```

Comment:
You'll need to instantiate `Message` on every request, so please move `new Message()` to here.

Reply:
OK, thank you